services/QuillLMS/app/views/evidence/_first_topic.html.erb

```diff
@@ -2,5 +2,5 @@
   <p class="topic">Culture & Society Topics</p>
   <img class="lazyload" alt="photograph of a football" loading="lazy" src="https://assets.quill.org/images/evidence/home_page/EvidenceLandingPage_Image_Football.webp" src="https://assets.quill.org/images/evidence/home_page/EvidenceLandingPage_Image_Football.webp"/>
   <p class="activity-title">"Should Schools Have Grade Requirements for Student Athletes?"</p>
-  <a href='https://www.quill.org/evidence/#/play?uid=180' target="_blank" class="q-button text-quillteal bg-white">View a sample activity</a>
+  <a href='https://www.quill.org/evidence/#/play?uid=180' target="_blank" class="q-button text-quill-teal bg-white">View a sample activity</a>
```

Comment:
I think this and the other instances of this class below need to be reverted or the name updated from `text-quillteal` to `text-quill-teal`

Reply:
@eadams17 thank you for flagging this! I think the class name has been updated here: https://github.com/empirical-org/Empirical-Core/blob/feat/replace-greens-with-variables-where-possible/services/QuillLMS/client/app/assets/styles/home.scss#L189 but let me know if I'm missing something.

Reply:
yeah, you got it! disregard-- it was just some hover states for buttons I was worried about, but looks like you got them covered here: https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/client/app/assets/styles/tool-section.scss#L24